eras/conway/test-suite/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-conway-test`
 
+## 1.2.1.0
+
+* Add `Crypto c` constraint to `exampleConwayGenesis`
```

Comment:
TIL that this kind of change has to be added to CHANGELOG too, makes sense.

Reply:
In a regular Haskell package this would also be a breaking change, since downstream users would be forced to add this constraint too, but no-one except us using this function and it can't really be used without Crypto constraint anyways, so this is no big deal and for us it is not really a breaking change. 
But, yes, the important part is that this is a semantic change, which should go into a changelog